app-connector/src/server/app.ts

```diff
@@ -0,0 +1,12 @@
+import * as connection from './connection';
+import { Event } from './event';
+import { API } from './api';
+
+let api = new API();
+
+let event = new Event();
+var runasync = async () => {
+    let all = await api.findAll();

```

Comment:
Why is that method call here?
Shouldn't we initialize the connection object here?

Reply:
removed it